### ISSUE TITLE
Accessibility: Multiple Non-UM Users progress indicator

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#645]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Critical

Feature: Add Multiple Non-UM Users

Issue notes:
Screen reader did not announce the progress indicator or indicate that is a part of "Select - Upload - Review - Confirmation"

<img width="1048" height="133" alt="Image" src="https://github.com/user-attachments/assets/7911ab9d-26b2-413b-bf96-7ef065966369" />

Criterion:
